android/src/toga_android/managersFileSystem.py

```diff
@@ -0,0 +1,29 @@
+class Singtools(type):
```

Comment:
I can't work out what this collection of utilities is trying to achieve, or why it is *almost* duplicated between the Android and Winforms backends.